### PR TITLE
Feature/modify meeting

### PR DESCRIPTION
**Pull request by beluramondelli**

### CODE REVIEW

File: Iteam-project/src/components/MymeetForm/MymeetForm.js [hunk 1]
```diff
@@ -79,25 +99,25 @@ class MymeetForm extends Component {
   showActions(meetingOwner, meetingDate) {
     if (this.isAdmin(meetingOwner)) {
       if (MymeetForm.validateDate(meetingDate)) {
-        return this.AdminUserActionsFinish;
+        return this.AdminActionsEdit;
```
Comment (jpressac) [inline Iteam-project/src/components/MymeetForm/MymeetForm.js:102]: The previous action was okey?. Add some comments in each if so could be understood easily.  😉

File: public-api/src/main/java/org/iteam/data/dal/meeting/MeetingRepositoryImpl.java [hunk 2]
```diff
@@ -33,198 +33,197 @@
...
-        return true;
+	private static final String IDEA_MEETING_ID_FIELD = "meetingId";
+	private static final String RANKING_ID_FIELD = "ranking";
+	private static final String MEETING_TEAM_NAME_FIELD = "teamName";
+	private static final String PROGRAMMED_DATE_FIELD = "programmedDate";
+	private static final int MAX_RETRIES = 5;
 
-    }
+	@Override
+	public boolean createMeeting(Meeting meeting) {
```
Comment (jpressac) [inline public-api/src/main/java/org/iteam/data/dal/meeting/MeetingRepositoryImpl.java:48]: You will need to update unit test. Go to MeetingRepositoryImpl line 315 (method givenAnElasticsearchClient) and change that line by this one

Mockito.when(elasticsearchClientImpl.insertData(Mockito.anyString(), Mockito.anyString(), Mockito.anyString(), Mockito.anyString()))
                .thenReturn(response);

And run the unitTest again.

File: Iteam-project/src/components/MymeetForm/MymeetForm.js (hunk 1 above)
Comment (beluramondelli) [reply Iteam-project/src/components/MymeetForm/MymeetForm.js:102]: Next pull

File: public-api/src/main/java/org/iteam/data/dal/meeting/MeetingRepositoryImpl.java (hunk 2 above)
Comment (beluramondelli) [reply public-api/src/main/java/org/iteam/data/dal/meeting/MeetingRepositoryImpl.java:48]: Done

**Pull request merged**